nautobot/circuits/views.py

```diff
@@ -97,7 +97,7 @@ def get_extra_context(self, request, instance):
             circuits = (
                 Circuit.objects.restrict(request.user, "view")
                 .filter(provider=instance)
-                .prefetch_related("type", "tenant", "terminations__site")
+                .prefetch_related("circuit_type", "tenant", "circuit_terminations__site")
```

Comment:
Gonna be a lot of merge conflicts between this and #3112, will need to handle those carefully.